.github/workflows/deploy-to-aks.yaml

```diff
@@ -0,0 +1,108 @@
+name: Create AKS cluster, deploy CKF and run bundle test
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
suggestion: have the k8s version as an input - with a default of 1.26
I can see the team wanting to test with different k8s versions, although it's not possible at the moment since 1.25 is no longer supported, but it will be useful for future versions of kubeflow. wdyt?